fe/fe-core/src/main/java/org/apache/doris/tablefunction/MetadataGenerator.java

```diff
@@ -1250,4 +1266,110 @@ private static TFetchSchemaTableDataResult tablePropertiesMetadataResult(TSchema
+        for (TableIf table : tables) {
+            if (!(table instanceof OlapTable)) {
+                continue;
+            }
+            if (!Env.getCurrentEnv().getAccessManager().checkTblPriv(currentUserIdentity, catalog.getName(),
+                    database.getFullName(), table.getName(), PrivPredicate.SHOW)) {
+                continue;
+            }
+
+            OlapTable olapTable = (OlapTable) table;
+            Collection<Partition> allPartitions = olapTable.getAllPartitions();
+
+            for (Partition partition : allPartitions) {
+                TRow trow = new TRow();
+                trow.addToColumnValue(new TCell().setStringVal(catalog.getName())); // TABLE_CATALOG
+                trow.addToColumnValue(new TCell().setStringVal(database.getFullName())); // TABLE_SCHEMA
+                trow.addToColumnValue(new TCell().setStringVal(table.getName())); // TABLE_NAME
+                trow.addToColumnValue(new TCell().setStringVal(partition.getName())); // PARTITION_NAME
+                trow.addToColumnValue(new TCell().setStringVal("")); // SUBPARTITION_NAME (not available)
+
+                trow.addToColumnValue(new TCell().setIntVal(0)); //PARTITION_ORDINAL_POSITION (not available)
+                trow.addToColumnValue(new TCell().setIntVal(0)); //SUBPARTITION_ORDINAL_POSITION (not available)
+                trow.addToColumnValue(new TCell().setStringVal(
+                        olapTable.getPartitionInfo().getType().toString())); // PARTITION_METHOD
+                trow.addToColumnValue(new TCell().setStringVal("")); // SUBPARTITION_METHOD
+                ArrayList<Expr> expr = olapTable.getPartitionInfo().getPartitionExprs();
+                String partExpr = (expr != null) ? olapTable.getPartitionInfo().getPartitionExprs().toString() : "";
+
+                trow.addToColumnValue(new TCell().setStringVal(partExpr)); // PARTITION_EXPRESSION (not available)
+                trow.addToColumnValue(new TCell().setStringVal("")); // SUBPARTITION_EXPRESSION (not available)
+                trow.addToColumnValue(new TCell().setStringVal(
+                        olapTable.getPartitionInfo().getPartitionRangeString(partition.getId())));
+                trow.addToColumnValue(new TCell().setIntVal(0)); //TABLE_ROWS (not available)
+                trow.addToColumnValue(new TCell().setIntVal(0)); //AVG_ROW_LENGTH (not available)
+                trow.addToColumnValue(new TCell().setIntVal(0)); //DATA_LENGTH (not available)
+                trow.addToColumnValue(new TCell().setIntVal(0)); //MAX_DATA_LENGTH (not available)
+                trow.addToColumnValue(new TCell().setIntVal(0)); //INDEX_LENGTH (not available)
+                trow.addToColumnValue(new TCell().setIntVal(0)); //DATA_FREE (not available)
+                trow.addToColumnValue(new TCell().setStringVal(
+                        TimeUtils.longToTimeString(partition.getVisibleVersionTime()))); //CREATE_TIME
```

Comment:
Actually, the `visible version time` is update time.
We don't have `create time`